src/components/street-geo.js

```diff
@@ -76,24 +77,36 @@ AFRAME.registerComponent('street-geo', {
     const data = this.data;
     const el = this.el;
 
-    const google3dElement = document.createElement('a-entity');
-    google3dElement.setAttribute('data-no-pause', '');
-    google3dElement.setAttribute('data-layer-name', 'Google 3D Tiles');
-    google3dElement.setAttribute('loader-3dtiles', {
-      url: 'https://tile.googleapis.com/v1/3dtiles/root.json',
-      long: data.longitude,
-      lat: data.latitude,
-      height: data.elevation - this.elevationHeightConstant,
-      googleApiKey: firebaseConfig.apiKey,
-      geoTransform: 'WGS84Cartesian',
-      maximumSSE: 48,
-      maximumMem: 400,
-      cameraEl: '#camera'
-    });
-    google3dElement.classList.add('autocreated');
-    google3dElement.setAttribute('data-ignore-raycaster', '');
-    el.appendChild(google3dElement);
-    return google3dElement;
+    const create3DtilesElement = () => {
+      const google3dElement = document.createElement('a-entity');
+      google3dElement.setAttribute('data-no-pause', '');
+      google3dElement.setAttribute('data-layer-name', 'Google 3D Tiles');
+      google3dElement.setAttribute('loader-3dtiles', {
+        url: 'https://tile.googleapis.com/v1/3dtiles/root.json',
+        long: data.longitude,
+        lat: data.latitude,
+        height: data.elevation - this.elevationHeightConstant,
+        googleApiKey: firebaseConfig.apiKey,
+        geoTransform: 'WGS84Cartesian',
+        maximumSSE: 16,
+        maximumMem: 400,
+        cameraEl: '#camera'
+      });
+      google3dElement.classList.add('autocreated');
+      google3dElement.setAttribute('data-ignore-raycaster', '');
+      el.appendChild(google3dElement);
+      return google3dElement;
+    };
+
+    // check whether the library has been imported. Download if not
+    if (AFRAME.components['loader-3dtiles']) {
+      return create3DtilesElement();
+    } else {
+      return loadScript(
+        new URL('/src/lib/aframe-loader-3dtiles-component.js', import.meta.url),
```

Comment:
does webpack automatically handle replacing this reference? the `/src/lib/` directory won't be available when deployed

Reply:
It's not done yet, sorry. I will change aframe-loader-3dtiles-component to another version and min version

Reply:
> does webpack automatically handle replacing this reference? the `/src/lib/` directory won't be available when deployed

But why not? I used the same path that is now in another place, in `AddLayerPanel/createLayerFunctions.js`

Reply:
Yes it might "just work" like the other one. We'll find out soon! :)

Reply:
Yes, webpack handles the resources defined as new URL('resource', import.meta.url), the resource will be minified and versionned (hash), that's why it appears as dist/48406973e3ad4d65cbdd.js

Reply:
See https://webpack.js.org/guides/web-workers/ about the URL handling.